### PR TITLE
Update odata.org to HTTPS

### PR DESCRIPTION
**Pull request by AlanWong-MS**
- Updating resource links to use HTTPS
- Addressing git warning regarding pointers in the assets folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Introduction
 
-This repository is the source code for [www.odata.org](http://www.odata.org) built with Jekyll. Site admins regularly updates the contents to the site.
+This repository is the source code for [www.odata.org](https://www.odata.org) built with Jekyll. Site admins regularly updates the contents to the site.
```
Comment (biaol-odata) [inline README.md]: We cannot support both through re-directing from HTTP to HTTPS?

**Comment by biaol-odata** [comment]
Not sure what git warning is related to the assert folder. Git does not keep track of folders AFAIK. Is that the  reason for changes related to the png files? No changes for the binary files themselve, right?

File: README.md (hunk 1 above)
Comment (AlanWong-MS) [reply README.md]: We can. The change for this specific file is optional. The required change is in _config.yml. I figured that I may as well update the popular pages to HTTPS while I am at it.

**Comment by AlanWong-MS** [comment]
As soon as you clone from master branch in the repo, you get a message regarding pointers. I am not able to perform `git checkout -- assets` and the change is still there.

image

**Comment by AlanWong-MS** [comment]
Did you clone from master prior to the merge?

https://github.com/OData/odataorg.github.io/tree/2fc865c64846331606ed22245174f132f8655d87

I cloned from the commit prior to the merge and the messages shows up still.
image

**Pull request merged**